kolibri/plugins/base.py

```diff
@@ -44,6 +46,15 @@ class KolibriPluginBase(object):
     # : Suggested property, not yet in use
     collect_static_on_enable = False
 
+    def __init__(self):
+        if settings.configured:
+            # Check to see if a plugin is being initialized after Django
+            warnings.warn(
```

Comment:
This warning helps to identify to the plugin author if their plugin is not being initialized at the right time in order for auto-discovered modules (such as settings and options) to have an effect on the operation of Kolibri.